lib/scrabble.rb

```diff
@@ -1,4 +1,60 @@
+POINTS = [
+  ["A", 1], ["E", 1], ["I", 1], ["O", 1], ["U", 1],
+  ["L", 1], ["N", 1], ["R", 1], ["S", 1], ["T", 1],
+  ["D", 2], ["G", 2],
+  ["B", 3], ["C", 3], ["M", 3], ["P", 3],
+  ["F", 4], ["H", 4], ["V", 4], ["W", 4], ["Y", 4],
+  ["K", 5],
+  ["J", 8], ["X", 8],
+  ["Q", 10], ["Z", 10]
+]
+
 module Scrabble
-  # require your gems and classes here
-  # require_relative 'scrabble/whatevs'
+  class Scrabble
+
+    # Returns the score of a word, also checking for non-letter characters
+    # If the word contains 7 characters 50 bonus points are rewarded
+    def self.score(word)
+      alphabet = %w(A B C D E F G H I J K L M N O P Q R S T U V W X Y Z)
+      word = word.upcase.chars
+      score = 0
+      (0..word.length).each do |index|
+        if alphabet.include?(word[index])
+          POINTS.each do |letter, point|
+            word.each do |word_letter|
```

Comment:
Very clever! For simplicity, however, you might consider using a Hash instead of an Array for the constant `POINTS`. This will allow you to access letters and their score directly, instead of having to do the work of matching up indices, etc. That is, `POINTS = { "A" => 1, "B" => 2 }` and so forth.